documentation/book/con-encryption-certificates.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: assembly-cluster-operator.adoc
+// * ID: [id='encryption-{context}']
+// * Title: = Encryption
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+include::common/attributes.adoc[]
+
+[id='certificates-{context}']
+= Certification Authority
+
+Each component needs its own private and public keys in order to support encryption.
+The public key has to be signed by a certificate authority (CA) in order to have a related X.509 certificate for providing server authentication and encrypting the communication channel with the client. The client could be another broker as well.
+All component certificates are signed by a Certification Authority (CA) called _cluster CA_.
+Another CA is used for authentication of Kafka clients connecting to Kafka broker and is called _clients CA_.
+The CAs themselves use self-signed certificates.
+
+The following table lists all the certificates that are generated and saved as Secrets in the {ProductPlatformName} cluster:
+
+.Security Certificates
+[cols="30%,50%",options="header"]
+|===
+|Certificate
+|Description
+
+|`_cluster-name_-cluster-ca`
```

Comment:
Is cluster-name a replaceable? if so put it in brackets here and all the way through the table.